docs/modules/IPFS.md

```diff
@@ -10,9 +10,9 @@ Returns a valid URL. If it is an IPFS URL (E.g. `ipfs://{hash}`), the IPFS Gatew
 
 #### Parameters
 
-| Name           | Type     | Description                                |
-| :------------- | :------- | :----------------------------------------- |
-| `url`          | `string` | The URL that is to be validated.           |
+| Name | Type | Description |
```

Comment:
What are your prettier settings? It looked better IMO the table before with the spacing between `|`.